aten/src/ATen/native/TensorIterator.cpp

```diff
@@ -479,10 +479,17 @@ std::unique_ptr<TensorIterator> TensorIterator::binary_op(Tensor& out, const Ten
   return builder.build();
 }
 
-std::unique_ptr<TensorIterator> TensorIterator::unary_op(Tensor& out, const Tensor& a) {
+std::unique_ptr<TensorIterator> TensorIterator::unary_op(Tensor& out, const Tensor& a, bool resize_outputs, bool different_types) {
```

Comment:
Use the builder parameter directly instead of adding extra parameters to the function. (This is going to get unwieldy very quickly, especially as we add more options to TensorIterator)

```
auto builder = TensorIterator::Builder();
builder.add_output(self);
builder.add_input(src);
builder.dont_compute_common_dtype();
builder.dont_resize_outputs();
```